From cd5cf1b569f660e8e811d5aa30cf4ee0046ea3c1 Mon Sep 17 00:00:00 2001
From: Markus Stockhausen <markus.stockhausen@gmx.de>
Date: Sun, 11 Aug 2024 08:50:49 -0400
Subject: [PATCH] realtek: backport gpio-realtek-otto driver from 5.13 to 5.10

This patch backports "gpio-realtek-otto" driver to Kernel 5.10.
"MACH_REALTEK_RTL" is used as a platform name in upstream, but "RTL838X"
is used in OpenWrt, so update the dependency by the additional patch.

Submitted-by: INAGAKI Hiroshi <musashino.open@gmail.com>
---
 drivers/gpio/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -527,8 +527,8 @@ config GPIO_RDA
 
 config GPIO_REALTEK_OTTO
 	tristate "Realtek Otto GPIO support"
-	depends on MACH_REALTEK_RTL
-	default MACH_REALTEK_RTL
+	depends on RTL83XX
+	default RTL838X
 	select GPIO_GENERIC
 	select GPIOLIB_IRQCHIP
 	help
